modules/db/src/main/java/org/mule/module/db/internal/processor/StoredProcedureMessageProcessor.java

```diff
@@ -30,13 +35,15 @@
 
 public class StoredProcedureMessageProcessor extends AbstractSingleQueryDbMessageProcessor
 {
+    protected final Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: "logger" is the name of a field in "AbstractInterceptingMessageProcessorBase". ([squid:S2387](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2387))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7720&issues=AWlIiAIYotIXnmsUxy1N&open=AWlIiAIYotIXnmsUxy1N)

[ISSUE_KEY]: AWlIiAIYotIXnmsUxy1N